### PR TITLE
[16.0][ADD] product_supplierinfo_purchase_contact: Add Purchase contact in product supplier info

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
This module adds Purchase Contact field on supplier rates (product.supplierinfo) and allows you to use this contact when creating the purchase order from supplies (replacement or to order).

@Tecnativa
TT49183

@pilarvargas-tecnativa @pedrobaeza

### CODE REVIEW

File: product_supplierinfo_purchase_contact/models/stock_rule.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                product_qty, product_id.uom_po_id
+            )
+            seller = product_id.with_company(company_id)._select_seller(
+                partner_id=partner,
+                quantity=uom_po_qty,
+                date=po.date_order and po.date_order.date(),
+                uom_id=product_id.uom_po_id,
+            )
+            res.update(self._prepare_purchase_order_line_from_seller(seller))
+        return res
```
Comment (chienandalu) [inline product_supplierinfo_purchase_contact/models/stock_rule.py]: This method doesn't belong to this  model (is from purchase.order.line). Is it needed at all?

Comment (carolinafernandez-tecnativa) [reply product_supplierinfo_purchase_contact/models/stock_rule.py]: Not necessary i just added to check if test failed, i just removed it

**Review by pilarvargas-tecnativa — APPROVED** [review]
Revised functionality, thanks

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**